icp-deploy.tf

```diff
@@ -61,14 +65,23 @@ module "icpprovision" {
       #"docker_password"                 = "${local.registry_creds != "" ? "${replace(local.registry_creds, "/.*:/", "")}" : "" }"
     }
 
+    hooks = {
+      "cluster-preconfig" = [
+        "sudo apt-get update",
```

Comment:
We'll want to consider RHEL/yum here as well. Or put in nicer failure logic when running on RHEL to skip this.